civihr_default_theme/scss/pages/_admin_absence_tab.scss

```diff
@@ -0,0 +1,47 @@
+.chr_leave-absences-admin {
+  background: transparent;
+  box-shadow: none;
+  padding: 0;
+}
+
+.chr_absence-tab-page {
```

Comment:
the design is totally different from what is there in my leave or manager leave, see the comments in css